db/repair.cc

```diff
@@ -554,10 +554,12 @@ class Repairer {
       ParsedInternalKey parsed;
       for (iter->SeekToFirst(); iter->Valid(); iter->Next()) {
         Slice key = iter->key();
-        if (ParseInternalKey(key, &parsed) != Status::OK()) {
+        Status pikStatus =
+            ParseInternalKey(key, &parsed, db_options_.allow_data_in_errors);
+        if (pikStatus != Status::OK()) {
           ROCKS_LOG_ERROR(db_options_.info_log,
                           "Table #%" PRIu64 ": unparsable key %s",
-                          t->meta.fd.GetNumber(), EscapeString(key).c_str());
+                          t->meta.fd.GetNumber(), pikStatus.getState());
```

Comment:
Just asking if we are confident the EscapeString part is no longer necessary?

Reply:
The Slice.toString(hex=true) made in ParseInternalKey() results in key being displayed in Hex....